apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -1022,6 +1028,27 @@ export class RushConfiguration {
     return this._repositoryUrl;
   }
 
+  /**
+   * The default git branch of the repository. This helps "rush change" find the right remote to compare against.
+   */
+  public get repositoryDefaultBranch(): string {
+    return this._repositoryDefaultBranch;
+  }
+
+  /**
+   * The default git remote of the repository. This helps "rush change" find the right remote to compare against.
+   */
+  public get repositoryDefaultRemote(): string {
+    return this._repositoryDefaultRemote;
+  }
+
+  /**
+   * The default fully-qualified git remote of the repository. This helps "rush change" find the right remote to compare against.
+   */
+  public get repositoryDefaultFullyQualifiedRemote(): string {
```

Comment:
This should probably be `repositoryDefaultFullyQualifiedBranch` or `repositoryDefaultFullyQualifiedRemoteBranch`.

Reply:
fixed